template/runtime.mustache

```diff
@@ -116,6 +132,13 @@ export const COLLECTION_FORMATS = {
 
 export type FetchAPI = {{#typescriptThreePlus}}WindowOrWorkerGlobalScope{{/typescriptThreePlus}}{{^typescriptThreePlus}}GlobalFetch{{/typescriptThreePlus}}['fetch'];
 
+export interface Logger {
```

Comment:
You can import `ILogger` from `nchat-dev-common`. We use it everywhere and it should be available.

Reply:
I prefer to keep the generated code as simple as we can and to avoid adding dependencies except if it's not necessary